GenderPayGap.WebUI/Views/ManageOrganisations/OrganisationReportsTable.cshtml

```diff
@@ -0,0 +1,83 @@
+﻿@using GenderPayGap.Core.Classes
+@using GenderPayGap.WebUI.Models.Organisation
+@using GenderPayGap.Core
+@model GenderPayGap.WebUI.Models.Organisation.OrganisationReportsViewModel
+
+<table class="govuk-table">
+    <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">Year</th>
+            <th scope="col" class="govuk-table__header">Reporting requirement</th>
+            <th scope="col" class="govuk-table__header">Report status</th>
+        </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+        @{
+            foreach (KeyValuePair<int, OrganisationScopeForYear> scopeForYear in Model.OrganisationScopesByYear)

```

Comment:
Since this is just a `foreach`, could we just do it as a list of `OrganisationScopeForYear`? (would also make the generation of Model.OrganisationScopesByYear a little easier)

Reply:
How I didn't think of this, I have no idea 😅